tests/templates/test_layers.py

```diff
@@ -451,7 +451,7 @@ def test_random_layer_randomwires(self, n_subsystems):
                 seed=42,
             )
 
-        wires = [q._wires for q in rec.queue]
+        wires = [q._wires.tolist() for q in rec.queue]
```

Comment:
How come this one needs a `tolist()`, but it was removed in most other places?

Reply:
The test computes the mean of randomly picked wires, so we need the integer representations here...